span_test.go

```diff
@@ -21,10 +21,12 @@ var (
 
 // spanPairs holds a set of trace.SpanData and its corresponding conversion to a ddSpan.
 var spanPairs = map[string]struct {
-	oc *trace.SpanData
-	dd *ddSpan
+	options Options
```

Comment:
I'd like `spanPairs` to contain simply a set of OpenCensus spans and their matching Datadog counterparts. Within that context, `options` has no place here. Please undo the changes to this file. I will suggest an alternative in the next comment.